src/redux/navigation.js

```diff
@@ -0,0 +1,24 @@
+var keyMirror = require('keymirror');
+
+var Types = keyMirror({
+    SET_SEARCH_TERM: null
+});
+
+module.exports.navigationReducer = function (state, action) {
+    if(typeof state = 'undefined') {
```

Comment:
I think this might be causing the test error – should be a space after `if`, and also it should be `===`